### PR TITLE
[Update 8] - Add std-lib new features to the release note

### PR DESCRIPTION
**Pull request by ayeshLK**

### CODE REVIEW

File: downloads/swan-lake-release-notes/swan-lake-2201.8.0/RELEASE_NOTE.md [hunk 1]
```diff
@@ -78,6 +78,15 @@ To view bug fixes, see the [GitHub milestone for 2201.8.0 (Swan Lake)](https://g
 
 ### New features
 
+##### `soap` Package
```
Comment (dilanSachi) [inline downloads/swan-lake-release-notes/swan-lake-2201.8.0/RELEASE_NOTE.md:81]:
```suggestion
#### `soap` Package
```
Only 4 right?

Comment (ayeshLK) [reply downloads/swan-lake-release-notes/swan-lake-2201.8.0/RELEASE_NOTE.md]: Fixed with [2d43cbc1bc843446371dbf284c74bae411008443](https://github.com/ballerina-platform/ballerina-dev-website/pull/7660/commits/2d43cbc1bc843446371dbf284c74bae411008443)

File: downloads/swan-lake-release-notes/swan-lake-2201.8.0/RELEASE_NOTE.md [hunk 2]
```diff
@@ -78,6 +78,15 @@ To view bug fixes, see the [GitHub milestone for 2201.8.0 (Swan Lake)](https://g
 
 ### New features
 
+#### `soap` Package
+- Introduced the `soap` standard library package, which provides a client API to connect to SOAP enpoints.
```
Comment (praneesha) [inline downloads/swan-lake-release-notes/swan-lake-2201.8.0/RELEASE_NOTE.md:82]:
```suggestion
- Introduced the `soap` standard library package, which provides a client API to connect to SOAP endpoints.
```
Is it correct to use both `library` and `package` words? Should it be the `soap` package?

File: downloads/swan-lake-release-notes/swan-lake-2201.8.0/RELEASE_NOTE.md [hunk 3]
```diff
@@ -78,6 +78,15 @@ To view bug fixes, see the [GitHub milestone for 2201.8.0 (Swan Lake)](https://g
...
+
+#### `mqtt` Package
+- Introduced the `mqtt` standard library package, which provides an implementation to interact with message brokers using MQTT protocol.
```
Comment (praneesha) [inline downloads/swan-lake-release-notes/swan-lake-2201.8.0/RELEASE_NOTE.md:85]:
```suggestion
- Introduced the `mqtt` standard library package, which provides an implementation to interact with message brokers using the MQTT protocol.
```
Ditto.

File: downloads/swan-lake-release-notes/swan-lake-2201.8.0/RELEASE_NOTE.md [hunk 4]
```diff
@@ -78,6 +78,15 @@ To view bug fixes, see the [GitHub milestone for 2201.8.0 (Swan Lake)](https://g
 
 ### New features
 
+#### `soap` Package
```
Comment (praneesha) [inline downloads/swan-lake-release-notes/swan-lake-2201.8.0/RELEASE_NOTE.md:81]:
```suggestion
#### `soap` package
```
All titles should be in sentence case.

File: downloads/swan-lake-release-notes/swan-lake-2201.8.0/RELEASE_NOTE.md [hunk 5]
```diff
@@ -78,6 +78,15 @@ To view bug fixes, see the [GitHub milestone for 2201.8.0 (Swan Lake)](https://g
...
+- Introduced the `soap` standard library package, which provides a client API to connect to SOAP enpoints.
+
+#### `mqtt` Package
+- Introduced the `mqtt` standard library package, which provides an implementation to interact with message brokers using MQTT protocol.
+
+#### `java.jms` Package
```
Comment (praneesha) [inline downloads/swan-lake-release-notes/swan-lake-2201.8.0/RELEASE_NOTE.md:87]:
```suggestion
#### `java.jms` Package
```
Ditto.

File: downloads/swan-lake-release-notes/swan-lake-2201.8.0/RELEASE_NOTE.md [hunk 6]
```diff
@@ -78,6 +78,15 @@ To view bug fixes, see the [GitHub milestone for 2201.8.0 (Swan Lake)](https://g
...
+- Introduced the `java.jms` standard library package, which provides an implementation to interact with message brokers using JMS protocol.
```
Comment (praneesha) [inline downloads/swan-lake-release-notes/swan-lake-2201.8.0/RELEASE_NOTE.md:88]:
```suggestion
- Introduced the `java.jms` standard library package, which provides an implementation to interact with message brokers using the JMS protocol.
```

**Review by praneesha — CHANGES_REQUESTED** [review]

File: downloads/swan-lake-release-notes/swan-lake-2201.8.0/RELEASE_NOTE.md [hunk 7]
```diff
@@ -78,6 +78,15 @@ To view bug fixes, see the [GitHub milestone for 2201.8.0 (Swan Lake)](https://g
 
 ### New features
 
+#### `soap` package
+- Introduced the `soap` standard library package, which provides a client API to connect to SOAP endpoints.
+
+#### `mqtt` Package
```
Comment (ayeshLK) [inline downloads/swan-lake-release-notes/swan-lake-2201.8.0/RELEASE_NOTE.md:84]:
```suggestion
#### `mqtt` package
```

File: downloads/swan-lake-release-notes/swan-lake-2201.8.0/RELEASE_NOTE.md [hunk 8]
```diff
@@ -78,6 +78,15 @@ To view bug fixes, see the [GitHub milestone for 2201.8.0 (Swan Lake)](https://g
...
+- Introduced the `mqtt` standard library package, which provides an implementation to interact with message brokers using the MQTT protocol.
+
+#### `java.jms` Package
```
Comment (ayeshLK) [inline downloads/swan-lake-release-notes/swan-lake-2201.8.0/RELEASE_NOTE.md:87]:
```suggestion
#### `java.jms` package
```

File: downloads/swan-lake-release-notes/swan-lake-2201.8.0/RELEASE_NOTE.md (hunk 5 above)
Comment (ayeshLK) [reply downloads/swan-lake-release-notes/swan-lake-2201.8.0/RELEASE_NOTE.md]: Fixed with [a57393068de956e2d8ae4e43952c3a5514baaf5d](https://github.com/ballerina-platform/ballerina-dev-website/pull/7660/commits/a57393068de956e2d8ae4e43952c3a5514baaf5d)

**Review by praneesha — APPROVED** [review]

**Review by MohamedSabthar — APPROVED** [review]

**Pull request merged**